app/controllers/companies_controller.rb

```diff
@@ -4,9 +4,8 @@
 
 class CompaniesController < ApplicationController
   def index
-    @companies = Company.all
-      .select { |company| company.condition == 'company active' }
-      .map { |company| CompanyPresenter.new(company) }
+    params[:condition] = 'company active'
```

Comment:
why do you need this?

Reply:
When user open app in the browser, on the begin app do index action and there is no `condition` parameter set so app display all db elements. I was thinking to do default setting:

`@companies = CompaniesSelector.call(params.permit(:query, :stock, condition: 'company active'))`

instead of 

`params[:condition] = 'company active'`
`@companies = CompaniesSelector.call(params.permit(:query, :stock, condition: 'company active'))`

Reply:
But this:

@companies = CompaniesSelector.call(params.permit(:query, :stock, condition: 'company active'))

not work, I think that permit method set permit, but can't set default

Reply:
in the future probably will be another view for something else than `company active` ,
I know that with this `params[:condition] = 'company active'` , I could skip passing this parameter when trigger index action, but I think I should pass it anyway

Reply:
I thin good idea is to do:

`params[:condition] = 'company active' unless params[:condition]`

Reply:
And now I can skip this parameter when trigger index action

Reply:
I think I will leave them, what you think you would do?

Reply:
If I leave them then I'm clear later when work with code what I want to get by action with this parameter.

Reply:
And if I make any changes in controller, the action will not change.

Reply:
> on the begin app do index action and there is no condition parameter set so app display all db elements. I was thinking to do default setting

ok it seems that were overwriting that parameter before instead of setting the default one. But now I can see you updated the code. Anyway I understand that you want to have a default value. thanks for the explanation.